### PR TITLE
assignment-1 done

### PR DESCRIPTION
**Pull request by kunha100**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
Adding a code
## What did you learn from the changes you have made?
how to make prediction by using knn
## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
no
## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
Yes, the standardized training set didn't contain class, so it wouldn't show the result. I searched for how to combine class into the test set.
## Checklist
- [ ] I can confirm that my changes are working as intended

### CODE REVIEW

**Review by FanWang0216 — CHANGES_REQUESTED** [review]
Question 3, you should output a single best hyperparameter found by GridSearchCV.

**Comment by kunha100** [comment]
> Question 3, you should output a single best hyperparameter found by GridSearchCV.

I added the new line. Could you please check?

**Review by FanWang0216 — APPROVED** [review]
